modules/gapi/test/common/gapi_core_tests_inl.hpp

```diff
@@ -1266,6 +1266,31 @@ TEST_P(SqrtTest, AccuracyTest)
     }
 }
 
+TEST_P(WarpPerspectiveTest, AccuracyTest)
+{
+    cv::Point center{in_mat1.size() / 2};
+    cv::Mat rm = cv::getRotationMatrix2D(center, angle, scale);
+    cv::Mat transform_mat = (Mat_<double>(3,3) << rm.at<double>(0, 0), rm.at<double>(0, 1), rm.at<double>(0, 2),
+                                                  rm.at<double>(1, 0), rm.at<double>(1, 1), rm.at<double>(1, 2),
+                                                  0                  , 0                  , 1                 );
```

Comment:
Please avoid using of this initializer - it bloats generated binary code.
Consider using of `Matx33d` and its constructor directly.

Reply:
Fixed